src/api/apiCore.cpp

```diff
@@ -62,7 +64,7 @@ RGL_API rgl_status_t rgl_get_version_info(int32_t* out_major, int32_t* out_minor
 	return status;
 }
 
-void TapePlayer::tape_get_version_info(const YAML::Node& yamlNode)
+void TapeCore::tape_get_version_info(const YAML::Node& yamlNode, TapeState& tapeState)
```

Comment:
Minor: consider `tapeState` -> `state`. It's rather obvious from the context, and `playbackState` might be lengthy.